.analysis_options

```diff
@@ -1,2 +1,4 @@
 analyzer:
-  strong-mode: true
+  strong-mode:
+    implicit-casts: False
+    implicit-dynamic: False
```

Comment:
> Does it imply that `dynamic` annotations in `fromUri` and `prettyUri` should be also removed?

Yes.

> When some other package depends on `path` and has `implicit-dynamic: False` analysis option, DDC fails if `dynamic` annotations are omitted:

`--no-implicit-dynamic` is not on by default, which means it's not a requirement for the package ecosystem. Downstream packages enable it at their own risk.